shared/scss/views/_hero.scss

```diff
@@ -0,0 +1,35 @@
+.hero {
+    padding: 0 20px 15px 20px;
+    text-align: center;
+    position: relative;
+}
+
+.hero__icon {
+    @include icon_display();
+    margin-top: 15px;
+}
+
+.hero__title {
+    line-height: 17px;
+    display: block;
+    font-weight: bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin-top: 11px;
+    text-align: center;
+    width: 100%;
+    max-width: 100%;
+    font-size: 17px;
+}
+
+.hero__subtitle {
+    padding-top: 6px;
+    @include uppercase_label();
+}
+
+// TODO this needs a bigger icon
+.hero__close {
+    position: absolute;
+    left: 10px;
+    top: calc(50% - 20px);
+}
```

Comment:
This unifies any reused code between the hero section in the main popup view and in the subview.

I initially said I was going to do this in a separate PR, but then that would've been a lot of PRs flying about in a short period of time, so I decided against it.